src/Domain/Organization/Services/OrganizationService.php

```diff
@@ -60,8 +66,31 @@ public function create(OrganizationCreateDto $dto): Organization
         foreach ($dto->shipping_addresses as $shipping_address) {
             $this->organizationSavedAddressService->storeAddress($shipping_address, SavedAddressType::SHIPPING, $organization->getKey(), $forceDefault);
         }
+        if ($dto->import) {
+            $company_name = match (true) {
+                is_string($dto->billing_address->company_name) => $dto->billing_address->company_name,
+                is_string($dto->billing_address->name) => $dto->billing_address->name,
+                default => $organization->getKey(),
+            };
+
+            $user = $this->userService->create(UserCreateDto::from([
+                'email' => is_string($dto->contact_email) ? $dto->contact_email : $dto->billing_email,
+                'password' => Str::password(),
+                'name' => is_string($dto->creator_name) ? $dto->creator_name : ('Przedstawiciel ' . $company_name),
+            ]), false);
 
-        $this->consentService->syncOrganizationConsents($organization, $dto->consents);
+            $organization->users()->attach($user->getKey());
+
+            $consents = Consent::query()
```

Comment:
`$dto->consents` has the structure `[consent_id => boolean]`, here it will be an array of UUIDs, so it needs to be changed to the same structure as `$dto->consents`.